lib/pointer.js

```diff
@@ -264,7 +265,7 @@ function setValue (pointer, token, value) {
     }
   }
   else {
-    throw ono.syntax(`Error assigning $ref pointer "${pointer.path}". \nCannot set "${token}" of a non-object.`);
```

Comment:
What is `ono` doing - what do we lose by ditching it? Upstream might consider this breaking if ono provides anything beyond the basics.

Reply:
According to docs, `ono.syntax` is just an alias for `SyntaxError`.